src/PIL/Image.py

```diff
@@ -3968,7 +3968,9 @@ def load(self, data: bytes) -> None:
         self._data.clear()
         self._hidden_data.clear()
         self._ifds.clear()
-        if data and data.startswith(b"Exif\x00\x00"):
+        if data and data.startswith(b"Exif\x00\x00Exif\x00\x00"):
+            data = data[12:]
+        elif data and data.startswith(b"Exif\x00\x00"):
```

Comment:
```suggestion
        while data and data.startswith(b"Exif\x00\x00"):
```
This seems simpler to me.

Reply:
Done. Also added test for this problem as well